hazelcast/include/hazelcast/client/internal/nearcache/impl/store/NearCacheDataRecordStore.h

```diff
@@ -49,14 +49,12 @@ class NearCacheDataRecordStore
     static const int64_t REFERENCE_SIZE =
       sizeof(std::shared_ptr<serialization::pimpl::data>);
 
-    NearCacheDataRecordStore(const std::string& name,
-                             const client::config::near_cache_config& config,
+    NearCacheDataRecordStore(const client::config::near_cache_config& config,
                              serialization::pimpl::SerializationService& ss)
       : BaseHeapNearCacheRecordStore<K,
                                      V,
                                      serialization::pimpl::data,
-                                     record::NearCacheDataRecord>(name,
```

Comment:
Could we revert this change? As it is public method, we should not change the signature. We might use another way to suppress the warning.